middleware/session/store.go

```diff
@@ -87,9 +86,36 @@ func (s *Store) Get(c *fiber.Ctx) (*Session, error) {
 	return sess, nil
 }
 
+// getSessionID will return the session id from:
+// 1. cookie
+// 2. http headers
+// 3. query string
+func (s *Store) getSessionID(c *fiber.Ctx) string {
+	id := c.Cookies(s.sessionName)
+	if len(id) > 0 {
+		return id
+	}
+
+	if s.source == SourceHeader {
+		id = string(c.Request().Header.Peek(s.sessionName))
+		if len(id) > 0 {
+			return id
+		}
+	}
+
+	if s.source == SourceURLQuery {
+		id = c.FormValue(s.sessionName)
```

Comment:
this should be the wrong function or ? if we want to get it from the query parameters it should be c.Query(s.sessionName)

Reply:
Although `c.FormValue()` will search the value from query string first, 

but you are right, it should only search from query string.

Changed here to use `c.Query()`. Thanks you!

#### Ref
https://github.com/valyala/fasthttp/blob/v1.23.0/server.go#L1004-L1036